src/vs/workbench/contrib/terminal/browser/terminalTabbedView.ts

```diff
@@ -69,6 +69,7 @@ export class TerminalTabbedView extends Disposable {
 	constructor(
 		parentElement: HTMLElement,
 		@ITerminalService private readonly _terminalService: ITerminalService,
+		@ITerminalService private readonly _terminalInstanceService: ITerminalInstanceService,
```

Comment:
`@ITerminalInstanceService`, I think this being wrong would have broken it at runtime?

Reply:
whoops - didn't cause an issue locally. good catch